xml/System/Math.xml

```diff
@@ -6585,7 +6585,7 @@ The following example demonstrates how to use the <xref:System.Math.Round%28Syst
       </ReturnValue>
       <MemberValue>6.2831853071795862</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Represents the number of radians in one turn, specified by the constant, &#x3C4;.</summary>
```

Comment:
I think the tool made these conversions of special symbols to their HTML entities. If they still look good in the MS Docs preview link, I think we can leave them.

I think the reason why I made this change was because the tool sometimes would crash if it opened a file with a specific encoding, but it had an unsupported character (not sure how that was happening).

I'll probably have to revisit that code.

Reply:
The symbols are rendering fine in MS Docs with their HTML entities: https://review.docs.microsoft.com/en-us/dotnet/api/system.math.tau?view=net-5.0&branch=pr-en-us-4752